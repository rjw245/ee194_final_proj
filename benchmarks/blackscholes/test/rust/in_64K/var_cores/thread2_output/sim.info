{'cmdline': ['/h/rwood04/ee194/sniper/run-sniper',
             '-d',
             './thread2_output',
             '-n',
             '2',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../../rust/bs',
             '2',
             '../../../inputs/in_64K.txt'],
 'git_revision': '',
 'host': 'dell26',
 'pin_version': 71313,
 'rusage': (12327.324961,
            6.225053,
            121352,
            0,
            0,
            0,
            475764,
            0,
            0,
            4376,
            11416,
            0,
            0,
            0,
            256334,
            681027),
 'snipercmd': ['bash',
               '-c',
               '/h/rwood04/ee194/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/rwood04/ee194/sniper/lib/pin_sim -c /h/rwood04/ee194/sniper/config/base.cfg --general/total_cores=2 --general/output_dir=/h/rwood04/ee194/ee194_docs/final_project/ee194_final_proj/benchmarks/blackscholes/test/rust/in_64K/thread2_output --config=/h/rwood04/ee194/sniper/config/nehalem.cfg --config=/h/rwood04/ee194/sniper/config/gainestown.cfg --config=/h/rwood04/ee194/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../../rust/bs 2 ../../../inputs/in_64K.txt'],
 't_elapsed': 16942.498866081238,
 't_start': 1460954529.464561,
 'tracecmds': [],
 'user': 'rwood04',
 'vmem': 3399733248L}
