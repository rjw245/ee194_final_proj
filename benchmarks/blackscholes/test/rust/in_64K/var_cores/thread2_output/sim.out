                                     | Core 0      | Core 1     
  Instructions                       |  4219431412 |      241390
  Cycles                             |  2480547622 |  2480547622
  IPC                                |        1.70 |        0.00
  Time (ns)                          |   932536700 |   932536700
  Idle time (ns)                     |       28501 |   932446543
  Idle time (%)                      |        0.0% |      100.0%
Branch predictor stats               |             |            
  num correct                        |   147358283 |       25127
  num incorrect                      |     6809083 |        2661
  misprediction rate                 |       4.42% |       9.58%
  mpki                               |        1.61 |       11.02
TLB Summary                          |             |            
  I-TLB                              |             |            
    num accesses                     |   858616556 |       29909
    num misses                       |         227 |         103
    miss rate                        |       0.00% |       0.34%
    mpki                             |        0.00 |        0.43
  D-TLB                              |             |            
    num accesses                     |  2813630833 |       83999
    num misses                       |       28187 |         276
    miss rate                        |       0.00% |       0.33%
    mpki                             |        0.01 |        1.14
  L2 TLB                             |             |            
    num accesses                     |       28414 |         379
    num misses                       |         902 |         231
    miss rate                        |       3.17% |      60.95%
    mpki                             |        0.00 |        0.96
Cache Summary                        |             |            
  Cache L1-I                         |             |            
    num cache accesses               |   858616556 |       29909
    num cache misses                 |    20497213 |        1485
    miss rate                        |       2.39% |       4.97%
    mpki                             |        4.86 |        6.15
  Cache L1-D                         |             |            
    num cache accesses               |  2813631870 |       84452
    num cache misses                 |      108004 |        2288
    miss rate                        |       0.00% |       2.71%
    mpki                             |        0.03 |        9.48
  Cache L2                           |             |            
    num cache accesses               |    20605395 |        3806
    num cache misses                 |       41645 |        3038
    miss rate                        |       0.20% |      79.82%
    mpki                             |        0.01 |       12.59
  Cache L3                           |             |            
    num cache accesses               |       41833 |        3072
    num cache misses                 |       34470 |         854
    miss rate                        |      82.40% |      27.80%
    mpki                             |        0.01 |        3.54
DRAM summary                         |             |            
  num dram accesses                  |       35303 |           0
  average dram access latency (ns)   |       65.13 |         inf
  average dram queueing delay        |       11.70 |        0.00
  average dram bandwidth utilization |       0.00% |       0.00%
Coherency Traffic                    |             |            
  num loads from dram                |        3992 |         125
  num loads from dram cache          |           0 |           0
  num loads from remote cache        |           0 |           0
